outlook/src/taskpane/components/Log/Logger.tsx

```diff
@@ -27,21 +34,123 @@ class Logger extends React.Component<LoggerProps, LoggerState> {
         };
     }
 
-    private logRequest = (event) => {
+    private fetchAttachmentContent(attachment, index): Promise<any> {
+        return new Promise<any>((resolve) => {
+            if (attachment.size > SIZE_THRESHOLD_SINGLE_ELEMENT * 1024 * 1024) {
+                resolve({
+                    name: attachment.name,
+                    inline: attachment.isInline && attachment.contentType.indexOf("image") >= 0,
+                    oversize: true,
+                    index: index,
+                });
+            }
+            Office.context.mailbox.item.getAttachmentContentAsync(attachment.id, asyncResult => {
+                resolve({
+                    name: attachment.name,
+                    content: asyncResult.value.content,
+                    inline: attachment.isInline && attachment.contentType.indexOf("image") >= 0,
+                    oversize: false,
+                    index: index,
+                });
+            });
+        });
+    };
+
+    private logRequest = async (event): Promise<any> => {
         event.stopPropagation();
 
         this.setState({logged: 1});
-        Office.context.mailbox.item.body.getAsync(Office.CoercionType.Html, (result) => {
+        Office.context.mailbox.item.body.getAsync(Office.CoercionType.Html, async (result) => {
             const msgHeader = '<div>From: ' + Office.context.mailbox.item.sender.emailAddress + '</div><br/>';
-            const msgFooter = '<br/><div class="text-muted font-italic">Logged from <a href="https://www.odoo.com/documentation/user/crm/optimize/mail_client_extension.html" target="_blank">Outlook Inbox</a></div>';
+            const msgFooter = '<br/><div class="text-muted font-italic">Logged from <a href="https://www.odoo.com/documentation/14.0/applications/sales/crm/optimize/outlook_extension.html" target="_blank">Outlook Inbox</a></div>';
```

Comment:
Side question: shouldn't Logged from be translated ? You could have a mix of languages (depending on email language), but makes probably sense to translate our additional stuff anyway. What do you think ?

Reply:
I think that translating this is a good idea, since this will enable us to be consistent with the other warning messages.